src/components/Button/Button.js

```diff
@@ -2,9 +2,9 @@ import * as React from 'react';
 import {Text, StyleSheet, TouchableOpacity} from 'react-native';
 import PropTypes from 'prop-types';
 
-import {colors} from '../../utils/colors';
 import {fonts} from '../../utils/fonts';
 import dimen from '../../utils/dimen';
+import {COLORS} from '../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been updated to import `COLORS` from the `theme` module instead of `colors` from the `colors` module. This change is consistent with the PR's goal of refactoring color usage across the application.
```diff
- import {colors} from '../../utils/colors';
+ import {COLORS} from '../../utils/theme';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->